pycket/expand.py

```diff
@@ -48,12 +48,17 @@ def readfile_rpython(fname):
 
 current_racket_proc = None
 
-def expand_string(s, reuse=True, srcloc=True):
+def expand_string(s, reuse=True, srcloc=True, byteOption=False, tmpFileName=False):
```

Comment:
Could you make the Python code you added consistent with the rest of the code base and avoid camel case names? Everything else in Pycket uses underscore separated identifier names.